Source/HTTP/httpcall.cpp

```diff
@@ -496,7 +496,7 @@ try
         return E_INVALIDARG;
     }
 
-    if (call->traceCall) { HC_TRACE_INFORMATION(HTTPCLIENT, "HCHttpCallPerform [ID %llu] uri: %s", call->id, call->url.c_str()); }
+    if (call->traceCall) { HC_TRACE_INFORMATION(HTTPCLIENT, "HCHttpCallPerform [ID %llu] uri: %s", TO_ULL(call->id), call->url.c_str()); }
```

Comment:
> TO_ULL( [](start = 98, length = 8)

what is the impact if you don't include these casts?

Reply:
It causes trouble on some platforms as discussed on this thread: https://stackoverflow.com/questions/14073877/printf-and-llu-vs-lu-on-os-x
but probably hard to see in practice since the IDs start off small